packages/app/components/stack-modal/StackOrdersTable.tsx

```diff
@@ -87,62 +87,64 @@ const TableCowBody = ({
   cowOrders: CowOrder[];
 }) => {
   const chainId = 100; // @todo use context or useNetwork hook
```

Comment:
Not related but we have update this hardcoded value

Reply:
Yeah! Totally agree, we can do this on this PR if we get to merge https://github.com/SwaprHQ/stackly-ui/pull/139 first <3

Reply:
Done!